src/helperFunctions/install.py

```diff
@@ -250,7 +251,7 @@ def check_distribution(allow_unsupported=False):
         return 'fedora'
     msg = (
         f'Your Distribution ({distro.id()} {distro.version()}) is not supported. '
-        'FACT Installer requires Ubuntu 20.04/22.04/24.04, Debian 11/12 or compatible!'
+        'FACT Installer requires Ubuntu 22.04/24.04, Debian 12 or compatible!'
```

Comment:
Thinking on this, the `allow_unsupported` parameter should be reachable from the `install.py` via command line argument. Maybe that's an additional PR though.